flow/alerting/alerting.go

```diff
@@ -439,9 +468,14 @@ func (a *Alerter) LogFlowError(ctx context.Context, flowName string, err error)
 		logger.Warn("failed to insert flow error", slog.Any("error", err))
 		return
 	}
+
+	errorClassString := ""
+
 	var tags []string
 	if errors.Is(err, context.Canceled) {
 		tags = append(tags, string(shared.ErrTypeCanceled))
+		// TODO this is only set for context.Canceled, other types need to be added too
+		errorClassString = "context.Canceled"
```

Comment:
why can't this pass tag through?

Reply:
I'm thinking whether our current classification is enough. Currently it is based on technical error types, we mostly need better differentiators

Reply:
> we mostly need better differentiators

To determine whether they are customer facing alerts or not, the errorClass is just an initial draft for now